server/orders/orderController.js

```diff
@@ -18,7 +18,25 @@ appRoute.route('/').post(async function(req,resp)
     console.log(`${JSON.stringify(newOrder)}`);
     let result = await addOrder(newOrder)
     return resp.json(result)
-})
+});
+
+appRoute.route('/memberid')
```

Comment:
Nothing to fix here yet, but we will talk about the security later. If you can call this api with Postman without login, it means that any malicious user can try to guess the id and get the member information. (Just a reminder for us to discuss)

Reply:
yes we need to talk about that